server/plugin/api.go

```diff
@@ -481,8 +481,11 @@ func (p *Plugin) getConnected(w http.ResponseWriter, r *http.Request) {
 
 		// Inform the user once that private repositories enabled
 		if val == nil {
-			p.CreateBotDMPost(info.UserID, "Private repositories have been enabled for this plugin. To be able to use them you must disconnect and reconnect your GitHub account. To reconnect your account, use the following slash commands: `/github disconnect` followed by `/github connect private`.", "")
-
+			if config.ConnectToPrivateByDefault {
+				p.CreateBotDMPost(info.UserID, "Private repositories have been enabled for this plugin. To be able to use them you must disconnect and reconnect your GitHub account. To reconnect your account, use the following slash commands: `/github disconnect` followed by `/github connect`.", "")
+			} else {
+				p.CreateBotDMPost(info.UserID, "Private repositories have been enabled for this plugin. To be able to use them you must disconnect and reconnect your GitHub account. To reconnect your account, use the following slash commands: `/github disconnect` followed by `/github connect private`.", "")
+			}
```

Comment:
Both the messages seem similar. can we instead use a single message and then use
```
message := 'Private repo .... by %s';
fmt.Sprintf(message, '`/github connect private`')
fmt.Sprintf(message, '`/github connect`')
```